spec/concepts/school/create_spec.rb

```diff
@@ -57,7 +57,7 @@
 
     it 'returns the error message in the operation response' do
       response = described_class.call(school_params:)
-      expect(response[:error]).to match(/Error creating school/)
+      expect(response[:error].count).to eq(6)
```

Comment:
could we just add another step that checks the response contains at least one sensible thing, like a validation?

Reply:
Sure, done in [bd13bf4](https://github.com/RaspberryPiFoundation/editor-api/pull/270/commits/bd13bf4d52d8c60890cb4e875bac91133cb7a263)